### ISSUE TITLE
Link API methods to corresponding API docs

### ISSUE THREAD

**Issue by mnquintana** [atom/docs#47]
It would be awesome for API methods mentioned in prose to link to their API doc definitions. This has been a common workflow for me – I'll read along in the guide, hit a method, and wonder more specifically how it works and open a new tab to search for it in the API docs. This would just shorten that process a bit. :smile:

**Comment by benogle** [comment]
Yeah, this should definitely be implemented in all cases. It is in some cases, e.g. most of them on https://atom.io/docs/latest/hacking-atom-package-modifying-text have been linked. Would be nice to be automatic. Looks like the author needs to use the absolute url, which is cumbersome.
